src/main/java/com/arpnetworking/tsdcore/sinks/WavefrontSink.java

```diff
@@ -274,11 +291,13 @@ public Builder setIdFile(final String value) {
         }
 
         /**
-         * Set of dimension names to be passed through as a Wavefront annotation/tag.
+         * Should no longer be used; restriction of dimensions should be done in MAD and/or on entry to cluster-aggregator.
          *
+         * @deprecated Restriction of dimensions should be done in MAD and/or on entry to cluster-aggregator.
          * @param value Set of white listed dimension names
          * @return This instance of <code>Builder</code>
          */
+        @Deprecated
```

Comment:
Hm, well, I feel like it should be. As the comment says, this is not the right place to restrict dimensions. But, sure, I'll remove this.